pkg/services/store/object/sqlstash/sql_storage_server.go

```diff
@@ -679,6 +683,29 @@ func (s *sqlObjectServer) Search(ctx context.Context, r *object.ObjectSearchRequ
 	return rsp, err
 }
 
+// This supports browsing the entity folder view
+func (s *sqlObjectServer) listEntityKinds(ctx context.Context) (*object.ObjectSearchResponse, error) {
+	kinds := []string{}
+
+	rsp := &object.ObjectSearchResponse{}
+	user := store.UserFromContext(ctx)
+	query := "SELECT DISTINCT(kind) FROM object WHERE parent_folder_path LIKE '" + fmt.Sprintf("%d/%s", user.OrgID, models.ObjectStoreScopeEntity) + `%'`
+	err := s.sess.Select(ctx, &kinds, query)
+	if err == nil {
```

Comment:
nit, but this could return early to remove a level of nesting

Reply:
dooh -- also removed from this PR since it was really here to make a UI thing work in:
https://github.com/grafana/grafana/pull/58288